pages/1.12/networking/index.md

```diff
@@ -81,6 +82,23 @@ While both Marathon-LB and Edge-LB are designed for handling north-south ingress
 | zero-hop load balancing               |     X     |         |             |
 | No single point of failure         |     X     |         |             |
 
+# <a name="cluster-id"></a>Specifying a cluster identity for network connections
+The DC/OS networking component (`dcos-net`) supports setting a **cluster identity** option on a node for DC/OS cluster. By enabling this feature, you can prevent nodes from communicating across clusters when a node is moved from one cluster to another. This feature ensures that the nodes from a cluster has a unique identifier and prevents unauthorized "cross-talk" between clusters.
+
+To use thw cluster identity feature:
+1. Edit the `config.yml` file for each node in the cluster to add the `dcos_net_cluster_identity` configuration parameter.
+
+1. Set the parameter value to `true` to enable the use of a cluster identity.
```

Comment:
Should this bullet be also '1.'? Looks like a typo

Reply:
It is a markdown thing. If you view with the Display rich diff option (toggle between [<>] and [pg] icons), it shows steps (1 and 2)